gradle.properties

```diff
@@ -1,2 +1,2 @@
 org.gradle.jvmargs=-Xmx1g
-version=1.2.4
+version=1.2.5-SNAPSHOT
```

Comment:
Looks like you have an extra commit here from Ed - can you remove this commit from your branch history?